cdap-ui/package.json

```diff
@@ -13,46 +13,46 @@
     "url": "https://github.com/caskdata/cdap/cdap-ui"
   },
   "devDependencies": {
-    "bower": "^1.3.12",
-    "del": "^1.1.1",
-    "gulp": "^3.8.11",
-    "gulp-angular-templatecache": "^1.5.0",
-    "gulp-autoprefixer": "^2.1.0",
-    "gulp-babel": "^5.1.0",
-    "gulp-concat": "^2.5.2",
-    "gulp-if": "^1.2.5",
-    "gulp-jshint": "^1.9.2",
-    "gulp-less": "^3.0.1",
-    "gulp-livereload": "^3.8.0",
-    "gulp-load-plugins": "^0.8.1",
-    "gulp-minify-css": "^1.0.0",
-    "gulp-minify-html": "^1.0.4",
-    "gulp-ng-annotate": "^0.5.2",
-    "gulp-plumber": "^1.0.1",
-    "gulp-replace": "^0.5.3",
-    "gulp-rev": "^3.0.1",
-    "gulp-size": "^1.2.1",
-    "gulp-sourcemaps": "^1.5.2",
-    "gulp-uglify": "^1.1.0",
-    "gulp-util": "^3.0.4",
-    "gulp-wrapper": "^0.1.5",
+    "bower": "1.3.12",
```

Comment:
I have restored it to a state where building wouldn't be a problem (as it used to be). I believe we have been having issues with the beamer and build because of bower update. We might want to invest sometime to upgrade all node modules post 3.3 as it will be quite some work to test it.